test/NuGet.Core.Tests/NuGet.ProjectModel.Test/TargetFrameworkInformationTests.cs

```diff
@@ -623,6 +623,8 @@ private static void AssertEquality(bool expected, TargetFrameworkInformation lef
             {
                 leftSide.Should().NotBe(rightSide);
             }
+
+            AssertClone(expected, leftSide, rightSide);
```

Comment:
Both the methods names (AssertEquality and AssertHashCode) don't inform the caller that they are going assert the equality of clones also. Not a blocking comment.